src/core/server/graph/schema/schema.graphql

```diff
@@ -4255,6 +4255,37 @@ type Queues {
   rejector: Queue!
 }
 
+type NextUnseenCommentPayload {
+  """
+  commentID is the new comment the next comment ID to focus on.
```

Comment:
Can you update this to clarify and make it consistent with the other comments here to something more like `commentID is the next comment ID to traverse to.`

Reply:
Updated!